editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/globalconfiguration/beans/BeanConfigUtil.java

```diff
@@ -668,4 +702,37 @@ public void editBeanProperty(Element xmlElement, String name, String value) {
 			xmlElement.setAttribute(GlobalBeanEIP.PROP_VALUE, value);
 		}
 	}
+	
+	public String getClassNameFromReferencedCamelBean(AbstractCamelModelElement selectedEP, String refID) {
+		if (!Strings.isEmpty((String) refID) && selectedEP != null) {
+			Map<String, GlobalDefinitionCamelModelElement> globalDefs = 
+					selectedEP.getCamelFile().getGlobalDefinitions();
+			GlobalDefinitionCamelModelElement referencedBean = globalDefs.get(refID);
+			if (referencedBean instanceof CamelBean) {
+				return ((CamelBean) referencedBean).getClassName();
+			}
+		}
+		return null;
+	}
+	
+	public String[] removeStringFromStringArray(String[] input, String deleteMe) {
+		ArrayList<String> result = new ArrayList<>();
```

Comment:
I prefer to use List everytime even if it is inside a method, I think it is shorter and easier to read and help to get good habits to avoid to return an ArrayList in an API